lib/mayaUsd/ufe/ProxyShapeHierarchy.cpp

```diff
@@ -332,10 +332,23 @@ ProxyShapeHierarchy::reorderCmd(const Ufe::SceneItemList& orderedList) const
 
 Ufe::SceneItem::Ptr ProxyShapeHierarchy::defaultParent() const
 {
-    // The default parent is the root of the USD scene.
-    // Since the proxy shape corresponds to the USD root prim,
-    // therefore we implement the default parent as the proxy shape itself.
-    return UsdSceneItem::create(sceneItem()->path(), getUsdRootPrim());
+    // The documentation for defaultParent() stipulate that it should return
+    // where this node should be inserted to be added back. The proxy shape need
+    // to be inserted under its Maya shape node, which is its default parent,
+    // so we return that.
+    //
+    // It used to return the USD virtual root prim, but that caused problem since
+    // the UFE path pointed to a Maya node (the proxy shape) and teh default parent
```

Comment:
Minor typo = "teh" -> "the" - don't bother re-running a preflight just for this.